openmetadata-docs/content/v1.5.x/connectors/database/mysql/index.md

```diff
@@ -7,8 +7,8 @@ slug: /connectors/database/mysql
 name="MySQL"
 stage="PROD"
 platform="OpenMetadata"
-availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "View Lineage", "View Column-level Lineage"]
-unavailableFeatures=["Query Usage", "Owners", "Tags", "Stored Procedures"]
+availableFeatures=["Metadata", "Data Profiler", "Data Quality", "dbt", "View Lineage", "View Column-level Lineage, "Query Usage"]
```

Comment:
quotes are not closed for  `"View Column-level Lineage`

Reply:
fixed